### PR TITLE
rebase: final changes  (8/8)

### PR DESCRIPTION
**Pull request by jaideepr97**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

> /kind bug
> /kind chore
> /kind cleanup
> /kind failing-test
> /kind enhancement
> /kind documentation
> /kind code-refactoring

**What does this PR do / why we need it**:
8/8 PRs to rebase the redesgin branch

This PR does:

- final misc changes to complete rebase

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Review by svghadi — APPROVED** [review]
lgtm

File: controllers/argocd/util.go [hunk 1]
```diff
@@ -36,7 +36,6 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	v1 "k8s.io/api/rbac/v1"
```
Comment (ciiay) [inline controllers/argocd/util.go:38]: Do we want to keep it `v1` or update it to `rbacv1` as imports in other files?

**Review by ciiay — APPROVED** [review]
LGTM
Just a small comment. Thank you.

File: controllers/argocd/util.go (hunk 1 above)
Comment (jaideepr97) [reply controllers/argocd/util.go:38]: `util.go` will eventually go away so we can leave it as is

**Pull request merged**